### PR TITLE
Show field types in tooltips

### LINKED ISSUE #3058: Tooltips don't show field types
If one hovers over a field, only the name is displayed in the tooltip:

image

For properties this works correctly:

image

### PR DESCRIPTION
**Pull request by xuanduc987**
Fixes #3058 

field

### CODE REVIEW

File: src/fsharp/symbols/SymbolHelpers.fs [hunk 1]
```diff
@@ -1068,6 +1068,8 @@ module internal SymbolHelpers =
                 NicePrint.layoutILTypeRef denv finfo.ILTypeRef ^^
                 SepL.dot ^^
                 wordL (tagField finfo.FieldName) ^^
+                RightL.colon ^^
+                NicePrint.layoutTyconRef denv (tcrefOfAppTy g finfo.EnclosingType) ^^
```
Comment (vasily-kirichenko) [inline src/fsharp/symbols/SymbolHelpers.fs]: I'm not sure that it's correct. I suspect EnclosingType is not we should use here, doesn't it refer to DateTime for any field defined in DateTime struct?

Comment (xuanduc987) [inline src/fsharp/symbols/SymbolHelpers.fs]: Yup, using `Enclosingtype` here is wrong.
Changed to `FieldType`

**Comment by dsyme** [comment]
@xuanduc987 This is great - ideally you should add a test too but I think we would accept it without one.

**Comment by KevinRansom** [comment]
Thanks for this contribution.  If you could submit an additional PR with some testing that would ensure we don't inadvertently break this in the future.

Thanks

Kevin

**Pull request merged**